packages/lib-user/src/components/UserHome/components/RecentProjects/RecentProjectsContainer.js

```diff
@@ -0,0 +1,44 @@
+import { shape, string } from 'prop-types'
+
+import { usePanoptesProjects, useStats } from '@hooks'
+import RecentProjects from './RecentProjects.js'
+
+export default function RecentProjectsContainer({ authUser }) {
+  // Returns all projects a user has classified in descending order (most classifications first)
+  const { data: statsData } = useStats({
+    sourceId: authUser.id,
+    query: { project_contributions: true }
+  })
```

Comment:
@mcbouslog @seanmiller26 as mentioned in the Slack thread, the response from this is a list of projects in descending order from greatest number of classifications to least number of classifications by the user.

Is this "Continue Classifying" section intended to be "projects you've most recently classified"? If so, I'll need to change this fetch to a user's project preferences like PFE's homepage.

Reply:
Yes, this is a recently classified section, so we can match PFE here.

Reply:
Sounds good - I refactored to display projects a user has most recently classified on. The catch with relying on `project_preferences` is that `project_preferences.updated_at` is changed when a user interacts with a project beyond just classifying. For instance, closing a tutorial. 

In this PR, I only fetched 1 page of `project_preferences` (20 items) and filtered for those where `activity_count > 0` (which means a user has actually classified on a project). This means if I recently interacted with 20 projects, but only classified on 3, then only 3 projects will show in this "Continue Classifying" UI section.

It would be much smoother if the eras request for `project_contributions` included a sort by most recently classified, but I'll have to follow-up with Michelle after internal team testing.

Reply:
A clarification - PFE's homepage recursively gets ALL pages of a user's project preferences rather than just 1 page. This is why it takes a long time for all of your ribbon classifications to load. I'd prefer not to use recursion on the new homepage because it's so data intensive - hence the note about follow-up with Michelle and eras.